charmcraft/commands/store/store.py

```diff
@@ -337,7 +337,7 @@ def list_revisions(self, name):
         return result
 
     @_store_client_wrapper()
-    def release(self, name, revision, channels, resources):
+    def release(self, name: str, revision: int, channels: List[str], resources):
```

Comment:
Yes this is very incomplete, but it improved it when I was finding issues.

This is probably going away in favour of using craft_store's upstream `release` eventually anyway.